app/api/custom/attendees.py

```diff
@@ -45,3 +48,50 @@ def send_receipt():
             return jsonify(message="receipt sent to attendees")
     else:
         raise UnprocessableEntityError({'source': ''}, 'Order identifier missing')
+
+
+@attendee_blueprint.route('/states', methods=['GET'])
+@jwt_required
+def check_attendee_state():
+    """
+    API to check attendee state is check in/registered
+    @return: user is registered or not
+    """
+    from app.models.event import Event
+
+    event_id = request.args.get('event_id')
+    attendee_id = request.args.get('attendee_id')
+    if event_id is not None:
```

Comment:
What behavior when args does not have any variable?
what behavior when event_id and attendee_id is None?
Consider this case.
```
if not len(request.args):
    raise ValueError("Could not find any variables")
if not request.args.get('event_id',False):
    raise ValueError("Missing Event id")
if not request.args.get('attendee_id',False):
    raise ValueError("Missing attendee id")
```

Reply:
fixed